samples/sensor/adxl362/src/main.c

```diff
@@ -22,8 +22,11 @@ static void trigger_handler(const struct device *dev,
 		}
 		k_sem_give(&sem);
 		break;
-	case SENSOR_TRIG_THRESHOLD:
-		printf("Threshold trigger\n");
+	case SENSOR_TRIG_INSTANT_MOTION:
+		printf("Activity trigger\n");
+		break;
+	case SENSOR_TRIG_STATIONARY:
+		printf("Inactivity trigger\n");
```

Comment:
Please align the printf strings to match the names of the triggers

Reply:
done